.vscode/bookmarks.json

```diff
@@ -0,0 +1,39 @@
+{
+	"files": [
+		{
+			"path": "vocab/credentials/v2/mk_vocab.rb",
+			"bookmarks": [
```

Comment:
We shouldn't commit .vscode files for personal developer bookmarks.

Reply:
Oops... I usually have this on .gitignore, but I missed it this time. Sorry about that.